system/jlib/jstats.cpp

```diff
@@ -2251,8 +2251,7 @@ void CRuntimeStatisticCollection::set(const CRuntimeStatisticCollection & other,
     {
         StatisticKind kind = other.getKind(i);
         unsigned __int64 value = other.getStatisticValue(kind);
-        if (value)
-            setStatistic(kind, value, node);
+        setStatistic(kind, value, node);
```

Comment:
I'm not sure this will ever be called - but that can wait until the refactor in https://track.hpccsystems.com/browse/HPCC-26541